validator/compareSwaggerYaml.py

```diff
@@ -0,0 +1,113 @@
+import ruamel.yaml as yaml
+import warnings as warnings
+import os
+import sys
+
+import constants
+
+global_issues = []
+
+
+def compare_definitions(yaml1, yaml2):
```

Comment:
Remember to add documentation for this file, not a Python expert, but a full example here https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html